resty/lib/src/interceptors.dart

```diff
@@ -13,7 +13,7 @@ abstract class Interceptor {
     return null;
   }
 
-  FutureOr<void> after(Response response);
+  FutureOr<dynamic> after(StringResponse response);
```

Comment:
must have the request object too to be able to retry the request

Reply:
Yes. Will fix that.